google/cloud/forseti/services/dao.py

```diff
@@ -1039,7 +1296,18 @@ def add_group_member(cls,
         def delete_group_member(cls, session, member_type_name,
                                 parent_type_name, only_delete_relationship,
                                 denorm=False):
-            """Delete member."""
+            """Delete member.
+
+            Args:
+                session(object): db session
+                member_type_name(str): type_name of the member to be deleted
+                parent_type_name(str): type_name of the parent if only delete
+                                       relation
```

Comment:
The convention is to do a hanging indent of 4-spaces:
```
    parent_type_name(str): type_name of the parent if only delete
        relation
```
Please fix this everywhere.